include/MVC/View/SugarView.php

```diff
@@ -1479,15 +1479,15 @@ public function getModuleTitle(
         if (SugarThemeRegistry::current()->directionality == "rtl") {
             $params = array_reverse($params);
         }
-        if (count($params) > 1) {
+        if (!$this->{'setModuleTitles'} && count($params) > 1) {
```

Comment:
Hi @MikeyJC, why are we using this notation `$this->{'setModuleTitles'}`?

Reply:
Hi, the parenthesis declares it as a property of `$this`, avoiding any notices regarding calling properties that don't exist yet.